azext_iot/central/providers/organization_provider.py

```diff
@@ -103,6 +108,7 @@ def create_organization(
             org_name=org_name,
             parent_org=parent_org,
             token=self._token,
+            update=update,
```

Comment:
what is the point of specifying update if you have a check that ensures the org_id does not exist yet?

Reply:
renamed to have "create_or_update" for better understandability.

We'll have a separate PR to remove all the checks like this "exist in" as we're going to remove all legacy caching in central commands.